pkg/controllers/management/auth/register.go

```diff
@@ -56,14 +56,14 @@ func RegisterIndexers(scaledContext *config.ScaledContext) error {
 }
 
 func RegisterEarly(ctx context.Context, management *config.ManagementContext, clusterManager *clustermanager.Manager) {
-	prtb, crtb := newRTBLifecycles(management)
-	gr := newGlobalRoleLifecycle(management)
-	grb := newGlobalRoleBindingLifecycle(management, clusterManager)
+	prtb, crtb := newRTBLifecycles(management.WithAgent("mgmt-auth-crtb-prtb-controller"))
+	gr := newGlobalRoleLifecycle(management.WithAgent(grController))
+	grb := newGlobalRoleBindingLifecycle(management.WithAgent(grbController), clusterManager)
 	p, c := newPandCLifecycles(management)
 	u := newUserLifecycle(management, clusterManager)
```

Comment:
I know the plan isn't to roll this out everywhere but I'm not clear why some of these were chosen and not others, for instance why do we want this for user attributes but not yet for users?